deeppavlov/dataset_iterators/sqlite_iterator.py

```diff
@@ -53,15 +51,34 @@ class SQLiteDataIterator(DataFittingIterator):
 
     """
 
-    def __init__(self, data_dir: str = '', data_url: str = DB_URL, batch_size: int = None,
+    def __init__(self, load_path: str, batch_size: int = None,
                  shuffle: bool = None, seed: int = None, **kwargs):
 
-        download_dir = expand_path(data_dir)
-        download_path = download_dir.joinpath(data_url.split("/")[-1])
-        download(download_path, data_url, force_download=False)
-
-        self.connect = sqlite3.connect(str(download_path), check_same_thread=False)
-        self.db_name = self.get_db_name()
+        if load_path is not None:
+            if load_path.startswith('http'):
+                logger.info("Downloading database from url: {}".format(load_path))
+                download_dir = expand_path(Path(load_path)).parent
```

Comment:
```python
expand_path(Path('http://files.deeppavlov.ai/datasets/wikipedia/enwiki.db')).parent
```
returns `PosixPath('/home/yoptar/reps/DeepPavlov/download/http:/files.deeppavlov.ai/datasets/wikipedia')` which is not ideal